### DISCUSSION TITLE
Tooling to debug serverless functions size

### DISCUSSION THREAD

**Discussion by alicescfernandes** [vercel/vercel#6513] [Ideas] [answered]
It would be cool if we could build the serverless functions locally and see the sizing of the built function. Lately i've been getting this `Error: The Serverless Function "api/geo.py" is 63.52mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size` error, and without a breakdown on what's causing this, it's hard to understand where the function is picking up this 63.52mb of.

**Comment by leerob** [comment, marked as answer]
Replaced by https://github.com/vercel/vercel/discussions/6516, please continue the discussion there 🙏
